tasks/main.yml

```diff
@@ -1 +1,17 @@
 # role tasks
+
+- name: Run shred to write zeros to the disk
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: 'shred -v -n 1 --random-source=/dev/zero -s \
+  "{{ disk_wipe_write_size }}" "{{ item }}"'
+  changed_when: true
```

Comment:
`changed_when` is redundant

Reply:
Added 301 to skip list and removed changed_when

Reply:
The skip list is global.  Since we are focused on just a couple of specific `command` tasks, we should just flag those specific tasks as ignored by the linter.

Reply:
OK added a tag to those to commands